swagger/specification/ucp/resource-manager/UCP/preview/2023-10-01-preview/examples/AwsPlanes_List.json

```diff
@@ -0,0 +1,24 @@
+{
+  "operationId": "AwsPlanes_List",
+  "title": "List AWS planes",
+  "parameters": {
+    "api-version": "2023-10-01-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
```

Comment:
Was it `values`?

Reply:
If it was, that would be wrong.

Reply:
https://github.com/radius-project/radius/pull/7499/files#r1591556066